### PR TITLE
Create ui-c8y-1019-20-8-disable-dashboard-availability-select-if-user-has-no-permission-to-request.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by jdre

Ticket: [MTM-57128](https://cumulocity.atlassian.net/browse/MTM-57128)
Original PR: [6173](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6173)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1019-20-8-disable-dashboard-availability-select-if-user-has-no-permission-to-request.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Disable dashboard availability select if user has no permission to request roles to populate it. (#6021) [GRAFT][release/cd] (#6173)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1019-20-8-disable-dashboard-availability-select-if-user-has-no-permission-to-request.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Disable dashboard availability select for users without permission to request roles
```

File: content/change-logs/application-enablement/ui-c8y-1019-20-8-disable-dashboard-availability-select-if-user-has-no-permission-to-request.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57128
+version: 1019.20.8
+---
+Disable dashboard availability select if user has no permission to request roles to populate it. (#6021) [GRAFT][release/cd] (#6173)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1019-20-8-disable-dashboard-availability-select-if-user-has-no-permission-to-request.md:17]: The Claude AI suggested to use the following body:
```suggestion
In the past, the dashboard availability select was always enabled for users, even if they did not have permission to request the roles required to populate it. With this change, the dashboard availability select is now disabled for users who do not have permission to request roles. This improves the user experience by avoiding confusion and clearly indicating to users when they are unable to use the dashboard availability select feature due to insufficient permissions.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by jdreSAG — APPROVED** [review]

**Review by jdreSAG — CHANGES_REQUESTED** [review]
get title and description from https://github.com/SoftwareAG/c8y-docs/pull/1965

File: content/change-logs/application-enablement/ui-c8y-1019-20-8-disable-dashboard-availability-select-if-user-has-no-permission-to-request.md (hunk 1 above)
Comment (jdreSAG) [inline content/change-logs/application-enablement/ui-c8y-1019-20-8-disable-dashboard-availability-select-if-user-has-no-permission-to-request.md:3]:
```suggestion
title: Dashboard availability selection is hidden for users who do not have the required permission
```

File: content/change-logs/application-enablement/ui-c8y-1019-20-8-disable-dashboard-availability-select-if-user-has-no-permission-to-request.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
```
Comment (jdreSAG) [inline content/change-logs/application-enablement/ui-c8y-1019-20-8-disable-dashboard-availability-select-if-user-has-no-permission-to-request.md:17]:
```suggestion
Previously, the dashboard availability selection was always visible for users, even if they did not have the permission to see it. With this change, the dashboard availability selection is hidden for users who do not have the required permission. Users must have User Management READ permission to see and modify the availability property of a dashboard.
```

**Review by jdreSAG — APPROVED** [review]

**Review by rahultsag — APPROVED** [review]

**Pull request merged**